scripts/validate_docstrings.py

```diff
@@ -473,8 +473,7 @@ def parameter_desc(self, param):
     @property
     def see_also(self):
         return collections.OrderedDict((name, ''.join(desc))
-                                       for name, desc, _
-                                       in self.doc['See Also'])
+                                       for name, desc in self.doc['See Also'])
```

Comment:
prob want to put a ``numpydoc>=0.9.0`` in environment.yml

alt you might be able to 

``for name, desc, * in self.doc[....]``